### PR TITLE
DOC-199 Reorg of Integrations Categories

### PR DESCRIPTION
**Pull request by janettelynch**
Added missing articles as well.

### CODE REVIEW

File: integrations.config.yaml [hunk 1]
```diff
@@ -234,34 +238,98 @@ integrations:
...
+    knowledgeCategoriesPaths: APIs_and-integrations/business-tools
   - projectName: graph-simplemdm
     displayName: SimpleMDM
     knowledgeCategoriesPaths: APIs_and-integrations/device-management
   - projectName: graph-freshservice
     displayName: Freshservice
     knowledgeCategoriesPaths: APIs_and-integrations/IT-service-management
+  - projectName: graph-cycognito
+    displayName: Cycognito
+    knowledgeCategoriesPaths: APIs_and-integrations/attach-surface-management
```
Comment (ninetreats) [inline integrations.config.yaml:274]: Looks like misspelling in /attack-surface-management

File: integrations.config.yaml [hunk 2]
```diff
@@ -234,34 +238,98 @@ integrations:
...
+    knowledgeCategoriesPaths: APIs_and-integrations/event-management
+  - projectName: graph-splunk-soar
+    displayName: Splunk SOAR
+    knowledgeCategoriesPaths: APIs_and-integrations/application-security
+  - projectName: graph-whois
+    displayName: Whois
+    knowledgeCategoriesPaths: APIs_and-integrations/cloud-infrastructure
+  - projectName: graph-atspoke
+    displayName: atSpoke
+    knowledgeCategoriesPaths: APIs_and-integrations/it-service-management
```
Comment (ninetreats) [inline integrations.config.yaml:319]: is this supposed to be IT? or it?

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-databricks.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
+# Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-databricks.md:1]: Should this be `Databricks` not Integration with JupiterOne

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-vsphere.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
+# VMware vSphere Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-vsphere.md:1]: Just VMware vSphere? not VMware vSphere Integration with JupiterOne

File: knowledgeBase/APIs_and-integrations/code/graph-esper.md [hunk 5]
```diff
@@ -1,112 +0,0 @@
-# Esper Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/code/graph-esper.md:1]: We still have an Esper integration.  Did you mean to delete this entire file?

File: knowledgeBase/APIs_and-integrations/customer-relationship-management/graph-hubspot.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
+# Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/customer-relationship-management/graph-hubspot.md:1]: Should this just be HubSpot (based on their website, looks like they capitalize the S in Spot.

File: knowledgeBase/APIs_and-integrations/endpoint-security/graph-microsoft-endpoint-defender.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
+# Microsoft Defender for Endpoint Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/endpoint-security/graph-microsoft-endpoint-defender.md:1]: delete the `Integration with JupiterOne text?

File: knowledgeBase/APIs_and-integrations/secret-management/graph-hashicorp-vault.md [hunk 8]
```diff
@@ -0,0 +1,175 @@
+# HashiCorp Vault Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/secret-management/graph-hashicorp-vault.md:1]: Delete the `Integration with JupiterOne` text?

**Review by ninetreats — COMMENTED** [review]
I added several comments in line on specific files.

File: knowledgeBase/APIs_and-integrations/application-development/graph-google-firebase.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
+# Google Firebase Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/application-development/graph-google-firebase.md:1]: delete `Integration with JupiterOne` text?

File: knowledgeBase/APIs_and-integrations/application-security/graph-probely.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
+# Probely Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/application-security/graph-probely.md:1]:
```suggestion
# Probely
```

File: knowledgeBase/APIs_and-integrations/application-development/graph-google-firebase.md (hunk 9 above)
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/application-development/graph-google-firebase.md:1]:
```suggestion
# Google Firebase
```

File: knowledgeBase/APIs_and-integrations/application-security/graph-sonarcloud.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
+# SonarCloud Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/application-security/graph-sonarcloud.md:1]:
```suggestion
# SonarCloud
```

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-databricks.md (hunk 3 above)
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-databricks.md:1]:
```suggestion
# Databricks
```

File: knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-vsphere.md (hunk 4 above)
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/cloud-infrastructure/graph-vsphere.md:1]:
```suggestion
# VMware vSphere
```

File: knowledgeBase/APIs_and-integrations/customer-relationship-management/graph-hubspot.md (hunk 6 above)
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/customer-relationship-management/graph-hubspot.md:1]:
```suggestion
# HubSpot
```
Based on their website, looks like they capitalize the S in their name.

File: knowledgeBase/APIs_and-integrations/device-management/graph-kandji.md [hunk 12]
```diff
@@ -1,26 +1,24 @@
-# Integration with JupiterOne
+# Kandji Integration with JupiterOne
```
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/device-management/graph-kandji.md:1]:
```suggestion
# Kandji
```
Should this just say Kandji?

File: knowledgeBase/APIs_and-integrations/endpoint-security/graph-microsoft-endpoint-defender.md (hunk 7 above)
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/endpoint-security/graph-microsoft-endpoint-defender.md:1]:
```suggestion
# Microsoft Defender for Endpoint

File: knowledgeBase/APIs_and-integrations/secret-management/graph-hashicorp-vault.md (hunk 8 above)
Comment (ninetreats) [inline knowledgeBase/APIs_and-integrations/secret-management/graph-hashicorp-vault.md:1]:
```suggestion
# HashiCorp Vault
```

File: knowledgeBase/APIs_and-integrations/code/graph-esper.md (hunk 5 above)
Comment (janettelynch) [reply knowledgeBase/APIs_and-integrations/code/graph-esper.md:1]: I moved it to device management.

File: knowledgeBase/APIs_and-integrations/application-security/graph-probely.md (hunk 10 above)
Comment (janettelynch) [reply knowledgeBase/APIs_and-integrations/application-security/graph-probely.md:1]: I will be doing all the title changes in another ticket.

File: knowledgeBase/APIs_and-integrations/application-security/graph-sonarcloud.md (hunk 11 above)
Comment (janettelynch) [reply knowledgeBase/APIs_and-integrations/application-security/graph-sonarcloud.md:1]: I will be doing all the title changes in another ticket.

**Review by ninetreats — APPROVED** [review]

**Pull request merged**